.gitignore

```diff
@@ -7,8 +7,3 @@ build
 .gradle
 .idea
 *.iml
-
-gradle/
-gradlew
-gradlew.bat
```

Comment:
We're using these now! Updating to a modern version of Gradle (2.2.1 -> 3.5 wrapper) buys us a ~3x speed increase.